### PR TITLE
sdf -> usd: Added sensor

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>

# 🎉 New feature

Parse SDF sensor and convert them in USD
 - Camera
 - Lidar
 - IMU

Added test too

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by adlarkin — COMMENTED** [review]
I made some changes in https://github.com/ignitionrobotics/sdformat/pull/830/commits/10d7da1c7c691c290221f1544b9aad3c1532323d and have one question, but other than that, LGTM!

File: usd/src/sdf_parser/Sensor.cc
Comment (ahcorde) [reply usd/src/sdf_parser/Sensor.cc:77]: Yep, they match, the only different is that Ignition uses radians and IssacSim uses degrees

**Review by adlarkin — APPROVED** [review]
@scpeters or @azeey, would you like to take a look at this before we merge it?

**Review by scpeters — APPROVED** [review]

**Comment by osrf-triage** [comment]
This pull request has been mentioned on **Gazebo Community**. There might be relevant details there:

https://community.gazebosim.org/t/new-ignition-releases-2022-04-13-fortress-edifice/1367/1

**Pull request merged**
